Create Millennia's portfolio home page

hilakey opened the pull request:
I would like feedback on my code, if the format style is okay.
- Added second page to portfolio.
- Added images for the gallery section.

Also, should anything else be added, or could be modified to suit the purpose of the first project?

hilakey committed:
Add images to second page for the Anime Recommendation Section.

hilakey committed:
Add images for pets on portfolio  home page.

hilakey committed:
Remove comments in code.

gagomez commented on portfolio/src/main/webapp/index.html:
```diff
@@ -4,15 +4,60 @@
+            <p> And I have a chihuahua named Sonic</p>
+            <a href="images/sonic.jpg"><img src="images/sonic.jpg" width="300" height="400"></a>
+            <a href="images/sonic2.PNG"><img src="images/sonic2.PNG" width="300" height="400"></a>
+        
```
The indenting here seems off. 

One thing you could do if you want to group these as a single element is you could wrap them in a `<div>`. For example:
```
<h2>Gallery</h2>
<div>
  <p>...
  <img>...
</div>
```

In a future stand up we can talk about components!

gagomez commented on portfolio/src/main/webapp/myportfoliopage2.html:
```diff
@@ -0,0 +1,57 @@
+        <a href="images/FF7R.jpg"><img src="images/FF7R.jpg" width="500" height="300" alt="Final Fantasy VII Remake"></a>
+        <p> Final Fantasy XIII Trilogy</p>
+        <a href="images/FF13.jpg"><img src="images/FF13.jpg" width="500" height="250" alt="Final Fantasy 13 Trilogy"></a>
+       <p> HALO</p>
```
Indentation is just a bit off. Off by one space.

hilakey replied:
Thank you for catching that! Just fixed the formatting.

hilakey committed:
Add button that randomly genereates a link from the Anime Recommendations list.

hilakey committed:
Provide alternative contents for images.
Make button that generates random link to recommended show.
Adjust HTML formatting.

sasymer commented on portfolio/src/main/webapp/script.js:
```diff
@@ -12,17 +12,22 @@
+
+    const animePick = animeList[Math.floor(Math.random() * animeList.length)];
+    window.open(animePick);
+
```
Maybe could get rid of these extra spaces

hilakey replied:
Done!

sasymer commented on portfolio/src/main/webapp/style.css:
```diff
@@ -1,9 +1,35 @@
+h1{
+    color: white;
+    background-color: pink;
+     border-width: 4px;
```
Indentation has one extra space.

hilakey replied:
Thank you Sanna, I just reformatted it!

sasymer reviewed (approved):
I left a couple comments just about spacing/indentation-looks good to me! Hopefully the comments will work and go through to you.

hilakey committed:
Omit type attribute for script.

hilakey committed:
Remove extra spacing in style.css file.

hilakey committed:
Remove extra indentations in script.js file.
